packages/adapters/fastify/web/src/web.ts

```diff
@@ -0,0 +1,179 @@
+import fs from 'fs'
+import path from 'path'
+
+import httpProxy from '@fastify/http-proxy'
+import fastifyStatic from '@fastify/static'
+import fastifyUrlData from '@fastify/url-data'
+import fg from 'fast-glob'
+import type { FastifyInstance, HookHandlerDoneFunction } from 'fastify'
+
+import { getConfig, getPaths } from '@redwoodjs/project-config'
+
+export interface RedwoodFastifyWebOptions {
+  redwood?: {
+    apiUrl?: string
+    apiUpstreamUrl?: string
```

Comment:
Not attached to this new name just know we need to change it

Reply:
To me it's difficult to understand what the difference is between `apiUrl` and `apiUpstreamUrl`.
But `apiHost` was also bad 🤷 

Maybe some doc comments are what we need 🤔 

"apiProxyTarget" – is that a better name? Or something else with "proxy" in the name?

Reply:
I prefer apiProxyTarget FWIW

Reply:
Yeah let's go with that

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9884/commits/99f5e35066f0b769f2625ec4f89598606a3e4296